src/manifest.rs

```diff
@@ -1,11 +1,14 @@
 use anyhow::{anyhow, Result};
 use k8s_openapi::apimachinery::pkg::apis::meta::v1::ObjectMeta;
 use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
 use std::convert::TryFrom;
 use validator::Validate;
 
 use policy_evaluator::policy_metadata::{Metadata, Rule};
 
+const POLICY_TITLE_ANNOTATION: &str = "io.kubewarden.policy.title";
```

Comment:
We should use https://github.com/kubewarden/policy-evaluator/blob/ff05e8aa55fbe69f0a4a2bc0dc638c47a9cc9ec1/src/constants.rs#L2